packages/create-app/index.js

```diff
@@ -44,6 +45,12 @@ async function init() {
       message: `Project name:`,
       initial: 'vite-project'
     })
+    if (!projectNameRE.test(name)) {
+      console.error(
+        `Project name "${name}" should only contain letters, numbers, underscores and dashes.`
```

Comment:
The rules are actually defined as the following: https://www.npmjs.com/package/validate-npm-package-name#naming-rules

Reply:
Yeah. I know what I submit is really simplified version, a subset of NPM package name. Do we really need so complicated rules like the NPM package name?

Reply:
I'd say just link to the spec,.

Why would you only want to support a subset of the rules? Valid package names are invalid only for the context of creating a Vite app? I'd say that would be a very poor move.

Reply:
@rschristian Hmm seems they'd better be the same. I'll modify it.